lib/nx_zlib.h

```diff
@@ -151,7 +156,8 @@ struct nx_config_t {
 	int      strategy_override; /** Force use of an specific deflate
 				     * strategy.  0 is fixed huffman, 1 is
 				     * dynamic huffman */
-	uint8_t  gzip_selector;
+	struct selector st_sel; /** strategy selector: selects between
```

Comment:
Now that we're using `{deflate,inflate}_mode`, this should be changed as well. To `mode_selector` or something similar. Same for the comment.

Reply:
@mscastanho How about calling it `mode` and removing the `_mode` from  `{deflate,inflate}_mode`?

Reply:
@rzinsly Sounds good.